ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/JsonLdPrettyWriter.java

```diff
@@ -1,18 +1,14 @@
 package be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter;
 
-import org.apache.jena.rdf.model.Model;
 import org.apache.jena.riot.RDFFormat;
 import org.apache.jena.riot.RDFWriter;
-
-import static be.vlaanderen.informatievlaanderen.ldes.ldi.rdf.formatter.PrefixAdder.addPrefixesToModel;
+import org.apache.jena.riot.RDFWriterBuilder;
 
 public class JsonLdPrettyWriter implements LdiRdfWriter {
 
 	@Override
-	public String write(Model model) {
-		return RDFWriter.source(addPrefixesToModel(model))
-				.format(RDFFormat.JSONLD10_PRETTY)
-				.asString();
+	public RDFWriterBuilder builder() {
```

Comment:
The LdiRdfWriter is a strategy pattern where different rdf formats have their own implementation. Given you have a specific rdf format -> binary, I wouldn't expect any changes in the jsonLdWriter